include/SFML/Window/Window.hpp

```diff
@@ -362,6 +363,25 @@ class SFML_WINDOW_API Window : GlResource, NonCopyable
     void setMouseCursorGrabbed(bool grabbed);
 
     ////////////////////////////////////////////////////////////
+    /// \brief Set the displayed cursor to a native system cursor
+    ///
+    /// Upon window creation, the arrow cursor is used by default.
+    ///
+    /// \warning The cursor must not be destroyed while in use by
+    ///          the window.
+    ///
+    /// \warning Features related to Cursor are not supported on
+    ///          iOS and Android.
+    ///
+    /// \param cursor Native system cursor type to display
+    ///
+    /// \see sf::Cursor::loadFromSystem
+    /// \see sf::Cursor::loadFromPixels
+    ///
+    ////////////////////////////////////////////////////////////
+    void setMouseCursor(const Cursor& cursor);
```

Comment:
> Did we discuss why we only have a setter and no getter?

I don't think we did. But in any case, we don't do it for the other properties (e.g. visibility, ...).

> We might want to document whether the cursor object can be discarded or needs to live on.

The doc reads:

> The cursor must not be destroyed while in use

and

> The behaviour is undefined if the cursor is destroyed while in use by the window.

;-)